packages/elements/src/types/page.ts

```diff
@@ -61,4 +61,9 @@ export interface Page {
   };
   autoAdvance?: boolean;
   defaultPageDuration?: number;
+  pageAttachment?: {
```

Comment:
While we're at it, there's also `shoppingAttachment` with `ctaText` and `theme` (no `url`)